### DISCUSSION TITLE
D878 v1 1.11n support

### DISCUSSION THREAD

**Discussion by mrma13** [hmatuschek/qdmr#342] [General]
I have an Anytone 878UV v1 (non-plus), 1.11N firmware and Sct 3258 Ver: V2.01.05NJ and qdmr v 0.11.1. 
Is this a supported combination?

EDIT: I just downloaded from the radio. I noticed the analog APRS Message  field contained repeating symbols instead of text.

**Comment by hmatuschek** [comment]
Firmware version 1.11n appears to be very old. The current one is 3.02n. There have been many changes in the codeplug structure since then. With QDMR, I try to keep up to date with the firmware version.

**Comment by mrma13** [comment]
I upgraded to 3.02n yesterday. I'm still fighting with Anytone's CPS to get everything working as before. After that I'll try qdmr again.
Thanks for the reply.

**Comment by mrma13** [comment]
Do you have a list of things qdmr does not program for the 878UV v1? ... such as the letter colors on the display?
Not a criticism. It easier knowing if something is not there rather than spending a lot of time looking for it.
I was able to DLl the codeplug from the radio, now 3.02n firmware. Lots of things are now populated that weren't before and those strange symbols that were in the APRS text field are now readable English.
Thanks

**Reply by allesand** [reply]
The current dev version has support for most of the device settings of the Anytone radios. Those are discussed in for example

https://github.com/hmatuschek/qdmr/issues/298

and

https://github.com/hmatuschek/qdmr/issues/340

The current stable version is missing mostly "device specific settings" as you noticed, but improvement is in the pipeline..

**Comment by mrma13** [comment]
Except for the color changes I don't think I use much of anything else mentioned so I haven't yet noticed them. I was looking for a list because that would help with figuring out if I messed up again or the feature was just not implemented yet.
Thanks for all the work.

**Comment by hmatuschek** [comment]
These settings will be part of the AnyTone device specific settings in the next release. By default, you will not see these settings. You will have to enable "Show device extensions" in the settings menu.

**Comment by mrma13** [comment]
That's interesting and specifically helpful to me.
Thanks for all the work.
